From 6bb058298bdd149288fc06ee6646f7343443951b Mon Sep 17 00:00:00 2001
From: Michaël Zasso <mic.besace@gmail.com>
Date: Tue, 23 Mar 2021 14:29:57 +0100
Subject: [PATCH] [wasm-simd] Add missing include for std::all_of

Fixes Node.js builds with GCC 8.

Change-Id: I3db574b48992f36ba42dec5c21d5c4b55dd7aa19
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/2780297
Auto-Submit: Michaël Zasso <mic.besace@gmail.com>
Commit-Queue: Jakob Kummerow <jkummerow@chromium.org>
Reviewed-by: Jakob Kummerow <jkummerow@chromium.org>
Cr-Commit-Position: refs/heads/master@{#73614}
---

diff --git a/v8/src/wasm/simd-shuffle.cc b/v8/src/wasm/simd-shuffle.cc
index 6260e8b..dac73a9 100644
--- a/v8/src/wasm/simd-shuffle.cc
+++ b/v8/src/wasm/simd-shuffle.cc
@@ -4,6 +4,8 @@
 
 #include "src/wasm/simd-shuffle.h"
 
+#include <algorithm>
+
 #include "src/common/globals.h"
 
 namespace v8 {
